keyboards/ergodox_ez/keymaps/rmw/keymap.c

```diff
@@ -0,0 +1,10 @@
+#include "ergodox_ez.h"
+#include "version.h"
+
+#define LONGPRESS_DELAY 180
+#define LAYER_TOGGLE_DELAY 350
+
+#include "tapdances.c"
+#include "keymap-mac.c"
+//#include "keymap-win.c"
+//#include "keymap-mw2.c"
```

Comment:
Sooo, what's up with all these keymaps?  
Why not just merge them?

Reply:
Fair enough. Someone on the SplitKB Discord needled me into starting a refactor anyway, so I'll merge them in that.